streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala

```diff
@@ -205,6 +205,33 @@ class StreamingContextSuite extends FunSuite with BeforeAndAfter with Timeouts w
     }
   }
 
+  test("stop slow receiver gracefully") {
```

Comment:
Yes, the current master branch fails this test. Exception because spark context shuts down before all jobs and only 10 out of 15 records gets produced (because of the 10 second thread join)